src/yashCJun20/Count.java

```diff
@@ -0,0 +1,47 @@
+package yashCJun20;
+
+public class Count {
+	
+	void count (String s){
+		
+		int digitCnt=0;
+		int upperCaseCnt=0;
+		int lowerCaseCnt=0;
+		int specialCharCnt;
+		
+		
+		for (int i=0;i<s.length();i++) {
+			char ch = s.charAt(i);
+			
+			if(Character.isDigit(ch)==true)
+				digitCnt++;
+			
+			if(Character.isUpperCase(ch)==true)
+				upperCaseCnt++;
+			
+			if(Character.isLowerCase(ch)==true)
+				lowerCaseCnt++;
```

Comment:
Here so many comparisons happening, which is not required. 
Like if char is digit so it will go in your all if block. which is wrong right !!
So here you need to write if, else if....else block.